src/index.html

```diff
@@ -11,7 +11,215 @@
+              id="Terms-input"
+              name="agreement"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email-input">
+            E-mail:
+            <input
+              type="email"
+              placeholder="email@example.com"
+              id="email-input"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="password-input">
+            Password:
+            <input
+              type="password"
+              name="Password"
+              id="password-input"
+              required
+              minlength="5"
+              maxlength="32"
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+          <label for="radio-input">
+
+            <input
+              type="radio"
+              name="cats"
+              id="radio-input"
+              value="Yes"
+            >
+            Yes
+          </label>
+
+          <label for="radio-input2">
+            <input
+              type="radio"
+              name="cats"
+              id="radio-input2"
+              value="No"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="color-input">
+            What is your favorite color?
+            <input
+              type="color"
+              id="color-input"
+              name="color"
+              value="#000"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="time-input">
+            What time do you go to bed?
+            <input
+              type="time"
+              name="time"
+              id="time-input"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="car-input">
+            What are you favorite brand of cars?
+            <select
+              id="car-input"
+              name="brand"
+              multiple
+            >
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="range-input">
+            How do you rate our work?
+            <input
+              type="range"
+              name="rate"
+              id="range-input"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>Additional info:</legend>
+        <div class="form-field">
+          <label for="comment-input">
+            Comments:
+            <textarea>
```

Comment:
you forgot to add `name` and `id` attributes for textarea

Reply:
oh yeah,thanks for the comment this